client/fresheyes-web/src/app/components/form/FormSection.tsx

```diff
@@ -44,8 +45,14 @@ const FormSection = () => {
 
     const res = client.processPullRequest(pr, (error, result) => {
       if (error) {
-        console.error(error);
-        setLoading({ loader: false, modal: false });
+        if (error.message.length === 0) {
+          setError("Opps an error occured you might want to check your setup");
```

Comment:
there's no setup required by the user. Let's just state that an error occurred. Please try again or contact support `info@bitcoindevs.xyz`

Reply:
Why I added this error message was when it returns a 404 error. The setup I meant here was ensuring the server was running